devguide/transactions.rst

```diff
@@ -339,7 +339,7 @@ So, for both privacy and security, we encourage you to build your applications t
 Transaction Malleability
 ------------------------
 
-None of Bitcoin’s signature hash types protect the signature script, leaving the door open for a limited denial of service attack called :term:`transaction malleability`\ {:.term}{:#term-transaction-malleability}. The signature script contains the `secp256k1 <http://www.secg.org/sec2-v2.pdf>`__ signature, which can’t sign itself, allowing attackers to make non-functional modifications to a transaction without rendering it invalid. For example, an attacker can add some data to the signature script which will be dropped before the previous pubkey script is processed.
+None of Bitcoin’s signature hash types protect the signature script, leaving the door open for a limited denial of service attack called :term:`transaction malleability <Transaction malleability>`\ {:.term}{:#term-transaction-malleability}. The signature script contains the `secp256k1 <http://www.secg.org/sec2-v2.pdf>`__ signature, which can’t sign itself, allowing attackers to make non-functional modifications to a transaction without rendering it invalid. For example, an attacker can add some data to the signature script which will be dropped before the previous pubkey script is processed.
```

Comment:
The "\ {:.term}{:#term-transaction-malleability}" part seems extraneous and can be removed. It shows up as-is in the rendered developer guide https://developer.bitcoin.org/devguide/transactions.html#transaction-malleability